docs/contributing/design-documents.rst

```diff
@@ -0,0 +1,74 @@
+:title: Design Documents
+:description: Information necessary for a feature design document.
+
+.. _design-documents:
+
+Design Documents
+================
+
+Before submitting a pull request which is expected to significantly change the
```

Comment:
I would rephrase this because I'm a picky old newspaper copy editor:
> Before submitting a pull request which will change the behavior of Deis significantly, such as a new feature or major refactoring, contributors...